stack.go

```diff
@@ -0,0 +1,150 @@
+package kama
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+func stackp() {
+	goModCache := os.Getenv("GOMODCACHE")
+
+	traces := getStackTrace()
+	for _, v := range traces {
+		if strings.Contains(v, "go/src/") {
+			// compiler
+			printWithColor(v, "cyan", false)
+		} else if goModCache != "" && strings.Contains(v, goModCache) {
+			// third party
+			printWithColor(v, "magenta", false)
+		} else if strings.Contains(v, "/home/komuw") { //TODO: fix this.
```

Comment:
@komuw fix this before merge

Reply:
fixed in https://github.com/komuw/kama/pull/29/commits/22e53d93d34338ab5d21ef48a7b6328f0d2107e8